tools/convert/README.md

```diff
@@ -22,7 +22,8 @@ Usage:
 	For config files, the new v2 config file is commented in detail to help explain what each
 	value does in the new configuration.
 
-	For example, if the v1 file specified "MetricsAPIKey" in the "HoneycombMetrics" section, the v2
+	For example, if the v1 file specifield has the same value as the default value, it will be commented out in the output file. Only non-default
+	value is not commented out.d "MetricsAPIKey" in the "HoneycombMetrics" section, the v2
```

Comment:
```suggestion
	Values that are the same as default values are commented out in the generated output file (since the default values are correct). Only values that do not match the default are enabled in the output file.
```